code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -206,7 +206,9 @@ datum
 
 			on_mob_life(var/mob/M, var/mult = 1)
 				if(!M) M = holder.my_atom
-				M.reagents.add_reagent("ethanol", alch_strength * mult)
+				M.reagents.add_reagent("ethanol", alch_strength * mult * depletion_rate)
+				//Multiplying by depletion rate makes alch_strength describe ABV, with 1 being 100% ABV
+				//This means that drinks ~15% ABV need a higher depletion rate so that the ethanol can accumulate
 				M.reagents.remove_reagent(src, 1 * mult)
```

Comment:
So it looks like this line was an attempt to solve something like this problem in the past, except it doesn't work because `remove_reagent` takes a reagent ID and not the reagent itself.
Since you've now solved it in a better way, you should probably remove this line!

Reply:
Yes, there is a reason for the depletion rate change! With the new alcohol strength change making everything 2.5x weaker, many of the drinks now add ethanol at a slower rate than Spacemen bodies purge it. To make it possible to get drunk off of these drinks, I upped the depletion rate.

Reply:
the line's been deleted, thanks for the tip!